### PR TITLE
AIW-3853 Flow bounty page refresh

### PR DESCRIPTION
**Pull request by aMacDawg**
deleted the flow bounty cards section

### CODE REVIEW

File: src/docs/automate-studio/flow-bounties/README.md [hunk 1]
```diff
@@ -172,320 +172,9 @@ If approved, Veritone will pay you the specified bounty amount for that flow. Th
...
-                <div class="description-text">This flow engine ingests the video in Youtube, transcribes it, and then runs topic extraction on that transcript. </div>
-            </div>
-            <a class="link-claimed" href="/#/automate-studio/flow-bounties/flow-bounty-details/youtube-scraper"> BUILD IT!</a>
-            <div class="bounty-number">Claimed</div>
-        </div>
-    </div>
-</div>
+## Automate Flow Wishlist 
+
+Check out our list of Automate Studio flow template projects and claim the flow template that is interesting for you to build! You can find the list of flows and claim one at a time with this [link](https://github.com/veritone/automate-flow-templates/tree/master/.github/ISSUE_TEMPLATE). <!-- {docsify-ignore} -->
```
Comment (kfiry) [inline src/docs/automate-studio/flow-bounties/README.md:177]: Wrong link. Should be: https://github.com/veritone/automate-flow-templates/projects/1
BUT ACTUALLY -- let's remove this section altogether. There are TWO more references to the wishlist on this page, both with the wrong link. Let's just fix the link for those two references, and then no need for this section anymore.

**Review by kfiry — CHANGES_REQUESTED** [review]

File: src/docs/automate-studio/flow-bounties/README.md (hunk 1 above)
Comment (aMacDawg) [reply src/docs/automate-studio/flow-bounties/README.md]: Got it, I updated the file with the correct url in the two spots and deleted the bottom section. Do we want to remove the sidebar references to each flow status as well?

Comment (kfiry) [reply src/docs/automate-studio/flow-bounties/README.md]: Yes to remove the sidebar links. 
And also, the "Get Started" link at the end is just sending back to the same page...

Comment (aMacDawg) [reply src/docs/automate-studio/flow-bounties/README.md]: thanks, fixed. Sidebar updated as well. I would like to merge [PR/343](https://github.com/veritone/docs/pull/343) in first

**Pull request closed without merging**